packages/worksheet/src/components/Actionsheet/WorksheetActionsheet.js

```diff
@@ -0,0 +1,202 @@
+                <IconByName
+                  name="AccountBoxFillIcon"
+                  _icon={{ size: 12 }}
+                  color="worksheetBoxText.400"
+                  p="0"
+                />
+                <Text
+                  fontWeight="400"
+                  fontSize="14px"
+                  color="worksheetBoxText.400"
+                >
+                  {"Grade: " + worksheet?.grade}
+                </Text>
+              </HStack>
+              <HStack space="1" alignItems="center">
+                <IconByName
+                  name="ArticleLineIcon"
+                  _icon={{ size: 12 }}
+                  color="worksheetBoxText.400"
+                  p="0"
+                />
+                <Text
+                  fontWeight="400"
+                  fontSize="14px"
+                  color="worksheetBoxText.400"
+                >
+                  {t("TOPIC") + ": " + worksheet?.topic}
+                </Text>
+              </HStack>
+              <HStack space="1" alignItems="center">
+                <IconByName
+                  name="Download2LineIcon"
+                  _icon={{ size: 12 }}
+                  color="worksheetBoxText.400"
+                  p="0"
+                />
+                <Text
+                  fontWeight="400"
+                  fontSize="14px"
+                  color="worksheetBoxText.400"
+                >
+                  {"Downloads: " + worksheet?.downloads}
+                </Text>
+              </HStack>
+            </VStack>
+          </HStack>
+          <HStack space={5} alignItems="center">
+            <HStack alignItems="center">
+              <IconByName
+                name="Heart3FillIcon"
+                color="red.500"
+                _icon={{ size: 12 }}
+                isDisabled
+              />
+              <Text fontWeight="600" fontSize="10px">
+                {`${likes.length} Teachers like this`}
+              </Text>
+            </HStack>
+            <Pressable onPress={(e) => handleCommentModuleOpen()}>
+              <HStack alignItems="center">
+                <Avatar.Group
+                  _avatar={{
+                    size: "md",
+                  }}
+                >
+                  <Avatar
+                    size="xs"
+                    bg="green.500"
+                    source={{
+                      uri: "https://images.unsplash.com/photo-1494790108377-be9c29b29330?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80",
+                    }}
+                  >
+                    AJ
+                  </Avatar>
+                  <Avatar
+                    size="xs"
+                    bg="cyan.500"
+                    source={{
+                      uri: "https://images.unsplash.com/photo-1603415526960-f7e0328c63b1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80",
```

Comment:
third party url used

Reply:
static data it will change in dynamic API